From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ThomasKaiser <github@kaiser-edv.de>
Date: Thu, 5 Jul 2018 05:32:09 -0700
Subject: [ARCHEOLOGY] Enable 1392 MHz cpufreq OPP on all RK3328 devices

> X-Git-Archeology: - Revision a79d5932cf21c1c7221bfa6cd036a3f47e506318: https://github.com/armbian/build/commit/a79d5932cf21c1c7221bfa6cd036a3f47e506318
> X-Git-Archeology:   Date: Thu, 05 Jul 2018 05:32:09 -0700
> X-Git-Archeology:   From: ThomasKaiser <github@kaiser-edv.de>
> X-Git-Archeology:   Subject: Enable 1392 MHz cpufreq OPP on all RK3328 devices
> X-Git-Archeology:
> X-Git-Archeology: - Revision 41b589372f2c1a8450033db78f3d8ed9db1b2196: https://github.com/armbian/build/commit/41b589372f2c1a8450033db78f3d8ed9db1b2196
> X-Git-Archeology:   Date: Sun, 26 Aug 2018 14:14:14 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Creating new family rockchip64. 3288 and 3399 was merged in, except Nanopi which stays in RK3399 family with sources attached to FA repository until we manage to join that in. Enabled Nanopc T4, Nanopi M4, Renegade. Tested on Rock64 and Nanopc T4.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3328.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
index e4debf4d6363..3ab51435101e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
@@ -165,6 +165,13 @@
 			opp-microvolt-L1 = <1300000 1300000 1350000>;
 			clock-latency-ns = <40000>;
 		};
+		opp-1392000000 {
+			opp-hz = /bits/ 64 <1392000000>;
+			opp-microvolt = <1350000 1350000 1350000>;
+			opp-microvolt-L0 = <1350000 1350000 1350000>;
+			opp-microvolt-L1 = <1325000 1325000 1325000>;
+			clock-latency-ns = <40000>;
+		};
 	};
 
 	arm-pmu {
-- 
Armbian

